lib/protobuf/message/fields.rb

```diff
@@ -12,8 +12,10 @@ def self.extended(other)
           :get_field_by_tag => :get_field,
         )
 
-        def inherited(subclass)
-          inherit_fields!(subclass)
+        other.class_eval do
```

Comment:
This is simply defining a class method on the extending class.  It can be moved to the ClassMethods like the other class methods we are defining on the extending class.